BlackJack_game/blackjack_simulate.py

```diff
@@ -0,0 +1,627 @@
+
+        self.recorder.draw_diagram()
+        ending = '\n\tSorry I lost all chips!\n\tTime to say goodbye.'
+        self.player.speak(ending)
+        print('\n' + '-' * 20 + ' End Game ' + '-' * 20)
+
+    def initial_game(self):
+        self.go_on = True
+        self.first_hand = True
+        self.choice = None
+        self.winner = None
+        self.bust = False
+        self.deck.rebuilt()
+        self.deck.shuffle()
+        self.player.chips.reset_chip()
+        self.player.drop_card()
+        self.player.refresh_prompt()
+        self.dealer.drop_card()
+        print('\n' + '-' * 20 + ' Start Game ' + '-' * 20)
+
+    def in_bet(self):
+        in_bet = '\n\tI want to bet: '
+        not_invalid = True
+        self.player.speak(in_bet, end_char='')
+        while not_invalid:
+            try:
+                self.player.chips.bet_amount = input()
+            except ValueError as e:
+                print(e)
+                self.player.speak(in_bet, end_char='')
+                continue
+            except KeyboardInterrupt:
+                print('')
+                self.recorder.draw_diagram()
+                quit()
+            else:
+                self.player.refresh_prompt()
+                # self.player.speak()
+                not_invalid = False
+
+    def deal_card(self):
+        # dealer
+        self.dealer.obtain_card(self.deck, face=False)
+        self.dealer.obtain_card(self.deck)
+
+        # player
+        self.player.obtain_card(self.deck)
+        self.player.obtain_card(self.deck)
+
+        self.dealer.showing()
+        self.player.showing()
+
+    def menu(self):
+        pattern = 'HS'
+        if self.first_hand:
+            pattern += 'U'
+            if (self.dealer.hand[1].rank == 1) and \
+                    self.player.chips.current_amount():
+                pattern += 'I'
+                self.dealer.ask_insurance()
+            if self.player.is_point('>', 10) and self.player.chips.can_double():
+                pattern += 'D'
+            self.first_hand = False
+        choices = self.player.select_choice(pattern)
+        select = self.get_select(len(choices),
+                                 general_err='Select above number.')
+        return choices[select]
+
+    @staticmethod
+    def get_select(select_max, prompt='>> ', general_err=''):
+        while True:
+            try:
+                value = input(prompt)
+                select = int(value)
+                if select > select_max:
+                    raise ValueError
+            except ValueError:
+                print(general_err)
+                continue
+            except KeyboardInterrupt:
+                print('')
+                quit()
+            else:
+                return select
+
+    def chips_manage(self):
+        if self.choice == 'Insurance':
+            err = 'The amount should under ' + \
+                  str(self.player.chips.current_amount())
```

Comment:
```suggestion
            err = ('The amount should under ' + \
                  str(self.player.chips.current_amount()))
```
As discussed in PEP8, try to avoid backslash line termination in Python code because whitespace to the right of the backslash will break the script on a change that is invisible to the reader.